okhttp/src/main/java/com/squareup/okhttp/MediaTypes.java

```diff
@@ -0,0 +1,108 @@
+package com.squareup.okhttp;
+
+/**
+ * Constants for commonly used media types.
+ *
+ * @see com.squareup.okhttp.MediaType
+ */
+public abstract class MediaTypes {
```

Comment:
I thought about putting the constants into the existing `MediaType` class but thought it might just blow up the code there.

Good point about the instantiation.